### PR TITLE
Feat/expense account eip712 frontend

### PR DESCRIPTION
**Pull request by aliassan**
# Description

Front end for EIP712 implementation of expense account.

https://www.loom.com/share/3d0484c8821c4906adf4ba25309dd6ff

Fixes # (issue)

## Contribution

For your PR please add a commnent to eache file edited to explain the changes you made.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist

Make sur you run the following commands before submitting your PR:

- /app

```bash
npm run build
npm run test:unit
npm run type-check
npm run lint
npm run format
```

- /backend

```bash
npm run build
npm run test
npm run lint
npm run format
```

- /contract

```bash
npm run test
npm run lint
npm run format
```

- **Please check if the PR fulfills these requirements**

- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added / updated (for bug fixes / features)

### CODE REVIEW

File: app/package.json
Comment (aliassan) [inline app/package.json:1]: Added datepicker component

File: app/src/components/sections/SingleTeamView/ExpenseAccountEIP712Section.vue
Comment (aliassan) [inline app/src/components/sections/SingleTeamView/ExpenseAccountEIP712Section.vue:1]: - Added EIP712 expense account info
- Added sign typed data to create signature and approval data structure to be save to the database and used when transferring from the smart contract

File: app/src/views/SingleTeamView.vue
Comment (aliassan) [inline app/src/views/SingleTeamView.vue:1]: - Use EIP712 expense account component

**Review by dasarathg68 — APPROVED** [review]
LGTM, please add tests and fix code quality check
🚀

**Pull request merged**